README.md

```diff
@@ -1,4 +1,4 @@
-# Kubernetes samples for IBM Cloud Container Service
-In this repository, you can find a collection of sample configuration scripts to deploy resources into Kubernetes clusters by using IBM Cloud Container Service.
+# Kubernetes samples for IBM Cloud Kubernetes Service
```

Comment:
Maybe we could change the heading to just "Samples for IKS" since now it says kube twice?